packages/shared/src/errors/Error.ts

```diff
@@ -69,6 +73,25 @@ export class ClerkAPIResponseError extends Error {
   };
 }
 
+export class ClerkRuntimeError extends Error {
```

Comment:
Given that we are adding new exports here, can we add JSDoc comments above `isClerkRuntimeError` and `ClerkRuntimeError` briefly explaining intended usage?